meta-balena-dunfell/README.md

```diff
@@ -0,0 +1,11 @@
+# Balena.io layer for Poky Warrior supported boards
```

Comment:
This README has a couple mentions of Warrior, as well as a mention of Sumo, should this be updated to Dunfell?

Reply:
agreed, will update that